stories/core/Table.stories.tsx

```diff
@@ -1324,3 +1327,117 @@ Full.args = {
 
 export const Condensed: Story<TableProps> = Basic.bind({});
 Condensed.args = { density: 'condensed' };
+
+export const Editable: Story<TableProps> = (args) => {
+  const { ...rest } = args;
+
+  type TableStoryDataType = {
+    name: string;
+    description: string;
+  };
+
+  const [data, setData] = React.useState(() => [
+    { name: 'Name1', description: 'Description1' },
+    { name: 'Name2', description: 'Description2' },
+    { name: 'Name3', description: 'Fetching...' },
+  ]);
+
+  const isRowDisabled = useCallback((rowData: TableStoryDataType) => {
+    return rowData.name === 'Name2';
+  }, []);
+
+  const onCellEdit = useCallback(
+    (columnId: string, value: string, rowData: TableStoryDataType) => {
+      action('onCellEdit')({ columnId, value, rowData });
+      setData((oldData) => {
+        const newData = [...oldData];
+        const index = oldData.indexOf(rowData);
+        const newObject = { ...newData[index] };
+        newObject[columnId as keyof TableStoryDataType] = value;
+        newData[index] = newObject;
+        return newData;
```

Comment:
This is weird... Interface expect cellEdit with void return, but here you return something.. does it have influence?

Reply:
Huh? It is a callback inside `setData`.

Reply:
oh crap... yeah, alignment didnt help :D